svgelements/svgelements.py

```diff
@@ -7713,12 +7735,18 @@ def __repr__(self):
             values.append("%s=%s" % (SVG_ATTR_WIDTH, Length.str(self.width)))
         if self.height != "100%":
             values.append("%s=%s" % (SVG_ATTR_HEIGHT, Length.str(self.height)))
-        if self.preserve_aspect_ratio is not None:
+        if self.image_width != 0:
+            values.append("image_width=%s" % Length.str(self.image_width))
+        if self.image_height != 0:
+            values.append("image_height=%s" % Length.str(self.image_height))
```

Comment:
The default image_width is `100%` not 0. 0 actually means 0 and should say it's zero.

Reply:
I will fix this too.

Reply:
These are not part of the SVG 2.0 spec, so I have removed.